renovate.json

```diff
@@ -12,6 +12,5 @@
       "matchPackageNames": ["graphene-django-optimizer", "django-timezone-field", "django-taggit"],
       "dependencyDashboardApproval": true
     }
-  ],
-  "rangeStrategy": "bump"
```

Comment:
@glennmatthews Per our discussion earlier, maybe `"update-lockfile"` is a better one to set to?


RE: https://docs.renovatebot.com/configuration-options/#rangestrategy

Reply:
Looks reasonable to me.